mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -346,9 +361,13 @@ def start_conn_process():
         if not callbacks__exit:
             self.logger.prn_wrn("missing __exit event from DUT")
 
-        #if not callbacks__exit and not result:
-        if not callbacks__exit and result is None:
-            self.logger.prn_err("missing __exit event from DUT and no result from host test, timeout...")
+        if not callbacks__exit_event_queue:
+            self.logger.prn_wrn("missing __exit_event_queue event from host test")
+
+        #if not callbacks__exit_event_queue and not result:
+        if not callbacks__exit_event_queue and result is None:
+            self.logger.prn_err("missing __exit_event_queue event from " + \
+                "host test and no result from host test, timeout...")
```

Comment:
Code clarity is important than meeting 80 char limit. I would review the log message rather than splitting it. May be: ```"Timeout as event `__exit_event_queue` missing"```